benchbuild/project.py

```diff
@@ -196,8 +199,8 @@ def __default_variant(self) -> VariantContext:
     group: str = attr.ib(
         default=attr.Factory(lambda self: type(self).GROUP, takes_self=True))
 
-    container = attr.ib(default=attr.Factory(lambda self: type(self).CONTAINER,
-                                             takes_self=True))
+    container = attr.ib(default=attr.Factory(
+        lambda self: copy.deepcopy(type(self).CONTAINER), takes_self=True))
```

Comment:
I don't know too much about `attr`, but can/should this be changed as we reassign this in the post-init anyway?

Reply:
fixed